src/routes/reservations_page.tsx

```diff
@@ -0,0 +1,59 @@
+import CollapsibleSection from 'components/collapsibleSection';
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import Section from 'components/section';
+import UnitItem from 'components/unitItem';
+import store from '../redux/store';
+import api from '../redux/api';
+
+export default function ReservationsPage() {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const ReservationsPage= () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry (I found 5):

 
![image](https://github.com/Elerqsousy/grandeur-frontend/assets/57421823/fe9fa4d3-cb2d-41f4-a741-0d748b1cc649)


Don't worry about the exports, as they can be done at the bottom of the file.

That way, your code will meet the latest standards. 💪